.github/workflows/functional-test.yaml

```diff
@@ -614,7 +640,7 @@ jobs:
           repo: ${{ github.repository }}
           sha: ${{ env.CHECKOUT_REF }}
           status: completed
-          conclusion: ${{ job.status }}
+          conclusion: ${{ needs.tests.outputs.test_status }}
```

Comment:
Maybe a dumb question about this about this ... does this work?

Since `tests` is a matrix are we going to end up with the same issue because each entry in the matrix will set `test_status`?

Would using `needs.tests.status` do the same thing? https://github.com/orgs/community/discussions/26822#discussioncomment-3253537